app/services/applier/workable_form_filler.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+module Applier
+  class WorkableFormFiller < FormFiller
+    def initialize(payload)
+      Capybara.configure { |config| config.test_id = :'data-ui' }
```

Comment:
What does this config do?

Reply:
You can use test_id to give Capybara an additional attribute to use as a locator (besides name and id).